versions.sh

```diff
@@ -159,7 +242,7 @@ for version in "${versions[@]}"; do
 				"3.19",
 				empty
 			| "alpine" + .),
-			if .arches | has("windows-amd64") and .["windows-amd64"].url then
+			if .arches | has("windows-amd64") and .["windows-amd64"].url then # TODO consider windows + tip
```

Comment:
It's technically _very_ doable for us to implement Windows support, but this was enough that I didn't want to get too much deeper and decided to leave Windows as a TODO for now instead.